### PR TITLE
Nc news 4

### PR DESCRIPTION
**Pull request by JackCollier**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -51,6 +51,32 @@ describe("GET /api/", () => {
...
+  });
+  test("should respond with an object article containing the passed id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(body).toHaveProperty("article", expect.any(Object));
+        expect(article).toMatchObject({
+          article_id: 1,
```
Comment (robert112233) [inline __tests__/api.test.js:66]: Well done for asserting that you get the specific Id back 👍

File: __tests__/api.test.js [hunk 2]
```diff
@@ -51,6 +51,32 @@ describe("GET /api/", () => {
...
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+});
+
```
Comment (robert112233) [inline __tests__/api.test.js:79]: You are missing some sad path tests for this endpoint: 

What happens if I make a request to /api/articles/notanid 

Would that be different to /api/articles/108489084 (doesn't exist) ?

File: models/api-models.js [hunk 3]
```diff
@@ -6,3 +6,11 @@ exports.selectTopics = () => {
     return rows;
   });
 };
+
+exports.selectArticleById = (article_id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then(({ rows }) => {
+      return rows[0];
```
Comment (robert112233) [inline models/api-models.js:14]: You may need a custom rejection here

**Review by Hard-Luck — COMMENTED** [review]
Just a small change then good to go!

**Pull request merged**